code/datums/diseases/advance/symptoms/blobspores.dm

```diff
@@ -57,7 +57,7 @@
 			ready_to_pop = TRUE
 			if(prob(5))
 				M.visible_message("<span class='warning'>[M] coughs blood!</span>")
-				new /obj/effect/decal/cleanable/blood/(M.loc)
+				M.add_splatter_floor(M.loc)
```

Comment:
this should also remove some blood

Reply:
Blood loss is not an intentional side effect of the symptom,  unless counting that caused by too much brute damage piling up

Reply:
I mean, it only makes sense to lose blood when you are coughing it